### PR TITLE
Update config templates to Juno

### PR DESCRIPTION
**Pull request by mjura**

### CODE REVIEW

File: chef/cookbooks/openstack-database/templates/default/trove-taskmanager.conf.erb [hunk 1]
```diff
@@ -85,11 +134,12 @@ dns_instance_entry_factory = trove.dns.designate.driver.DesignateInstanceEntryFa
 dns_endpoint_url = http://127.0.0.1/v1/
 dns_service_type = dns
 
+# Neutron
+network_driver = trove.network.nova.NovaNetwork
+default_neutron_networks =
```
Comment (vuntz) [inline chef/cookbooks/openstack-database/templates/default/trove-taskmanager.conf.erb]: Shouldn't this be set to something?

Comment (mjura) [inline chef/cookbooks/openstack-database/templates/default/trove-taskmanager.conf.erb]: No, according documentation and configuration sample for taskamanager, it is fine

**Comment by rsalevsky** [comment]
Is this PR still valid or did the config changed again in the meanwhile?

**Comment by mjura** [comment]
Yes, this PR is still valid and should be merged

**Comment by rsalevsky** [comment]
@mjura, this repository got merged into [crowbar-openstack](https://github.com/crowbar/crowbar-openstack).
Please open a new pull request there. __Do NOT merge here anymore!__

**Comment by mjura** [comment]
Not need to merge

**Pull request closed without merging**